app/views/users/show.html.erb

```diff
@@ -1,3 +1,13 @@
-<h1> Displays each users Details </h1>
 
-<p>Name: <strong><%= @user.name %> id: <%=@user.id %></strong></p>
+  <%= render partial: 'partials/user', locals: { user: @user } %>
+  <div class="bio">
+    <h3>Bio</h3>
+    <p><%=@user.bio%></p>
+  </div>
+  <div class="posts">
+    <% @user.posts.limit(3).each do |post| %>
```

Comment:
- [ ] Please consider using the method you created in an earlier milestone: 🤓 🔧 

https://github.com/ProgramKingAbel/tubongeApp/blob/eb0868d9b78f13de205bfbab241cd64ec1b4b71d/app/models/user.rb#L9-L11